platform/util/platform-util-unavailableurls/pom.xml

```diff
@@ -24,6 +26,11 @@
     <packaging>jar</packaging>
     <dependencies>
         <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
I can pull it out if it isn't in the best interest of the code base
I thought it was elegant, but it isn't necessary.